src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/Spinner.xaml.cs

```diff
@@ -3,21 +3,35 @@
 
 using System;
 using System.Collections.ObjectModel;
+using System.ComponentModel;
 using System.Globalization;
 using System.Linq;
+using System.Runtime.CompilerServices;
 using System.Windows.Controls;
 using System.Windows.Data;
 using System.Windows.Media;
 
 namespace NuGet.PackageManagement.UI
 {
-    internal class EllipseDetails
+    internal class EllipseDetails : INotifyPropertyChanged
     {
         public double Width { get; set; }
         public double Height { get; set; }
         public double Left { get; set; }
         public double Top { get; set; }
         public Brush Fill { get; set; }
+        
+        // Added INotifyPropertyChange to avoid memory leak in Spinner control. 
```

Comment:
Can you describe in a bit more detail how this helps?

Reply:
https://stackoverflow.com/questions/18542940/can-bindings-create-memory-leaks-in-wpf
Instead of CLR does binding for us, we explicitly tell how binding is done. So this way when window closes it can clean up better since it knows about exact binding. :)